### ISSUE TITLE
composer validation warnings

### ISSUE THREAD

**Issue by n4ss1m** [Chargily/chargily-pay-php#7]
Composer validation currently produces several warnings due to unbound version constraints in `composer.json`. 
These issues may impact package stability.

```bash
composer validate
```

Output : 

```
./composer.json is valid, but with a few warnings
See https://getcomposer.org/doc/04-schema.md for details on the schema
# General warnings
- require.guzzlehttp/guzzle : unbound version constraints (>=7.0) should be avoided
- require.rakit/validation : unbound version constraints (>=1.4) should be avoided
- require.illuminate/support : unbound version constraints (>=10.39) should be avoided
- require.medboubazine/number_format : unbound version constraints (>=1.1) should be avoided
- require.nesbot/carbon : unbound version constraints (>=2.72) should be avoided
```

**Comment by n4ss1m** [comment]
this issue should be fixed in this PR #6

**Comment by Medboubazine** [comment]
Thanks for your contribution
